bin/phoenix_utils.py

```diff
@@ -162,6 +163,13 @@ def setPath():
     if phoenix_queryserver_jar == "":
         phoenix_queryserver_jar = findFileInPathWithoutRecursion(PHOENIX_QUERYSERVER_JAR_PATTERN, os.path.join(current_dir, ".."))
 
+    global phoenix_loadbalancer_jar
+    phoenix_loadbalancer_jar = find(PHOENIX_LOADBALANCER_JAR_PATTERN, os.path.join(current_dir, "..", "phoenix-loadbalancer", "target", "*"))
+    if phoenix_loadbalancer_jar == "":
+        phoenix_loadbalancer_jar = findFileInPathWithoutRecursion(PHOENIX_LOADBALANCER_JAR_PATTERN, os.path.join(current_dir, "..", "lib"))
```

Comment:
I'm just noticing in my 5.1.0-SNAPSHOT build that the phoenix-load-balancer jar isn't there at all. Does phoenix-assembly need to be updated as well?

Reply:
@joshelser Load balancer is presented as a separate jar-file and the query-server.jar is uses for connection to load-balancer special java-class - ServiceLoader, which is looking for the right jar-file with implementation of necessary class. For release you just need to rebuild load-balancer.jar with my fix ("added service for registry in meta-inf"). For this you can use pom.xml in the load-balancer package.

Reply:
Yes, I understand how the load-balancer is meant to work (I was involved in the reviews and stipulated it be this way ;)).

I'm trying to point out that the logic you've added here is useless on master because the jar doesn't exist in the lib directory of the tarball created by the phoenix-assembly module. Specifically, the following path is never valid because the JAR is not put there on normal tarball builds of Phoenix
```
findFileInPathWithoutRecursion(PHOENIX_LOADBALANCER_JAR_PATTERN, os.path.join(current_dir, "..", "lib"))
```

Reply:
Oh, I understand, yeah, I'll fix it

Reply:
@joshelser ready :)